### PR TITLE
SYNCT-5: Added configuration UI for Sync 2.0

### PR DESCRIPTION
**Pull request by areklalo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -175,6 +175,8 @@
 		<javaVersion>1.8</javaVersion>
         <openmrsPlatformVersion>2.0.5</openmrsPlatformVersion>
 		<legacyVersion>1.3.1</legacyVersion>
+		<uiframeworkVersion>3.13.0</uiframeworkVersion>
```
Comment (pkornowski) [inline pom.xml]: Change UI framework version to 3.11.0

File: pom.xml [hunk 2]
```diff
@@ -175,6 +175,8 @@
...
+		<uicommonsVersion>2.4.0</uicommonsVersion>
```
Comment (pkornowski) [inline pom.xml]: Change the UI Commons Version to 2.1

File: pom.xml (hunk 1 above)
Comment (areklalo) [reply pom.xml]: Corrected.

File: pom.xml (hunk 2 above)
Comment (areklalo) [reply pom.xml]: Corrected.

**Comment by pkornowski** [comment]
Looks good now.

**Pull request merged**